fedimint-core/src/api.rs

```diff
@@ -154,21 +155,22 @@ pub trait IFederationApi: Debug + MaybeSend + MaybeSync {
 /// Notably the caling convention of fedimintd api is a bit weird ATM, so by
 /// using this function you'll make it easier to change it in the future.
 pub fn erased_no_param() -> Vec<JsonValue> {
-    vec![JsonValue::Null]
+    let params_raw = serde_json::to_value(ApiRequest::default())
+        .expect("parameter serialization error - this should not happen");
+    vec![params_raw]
 }
 
 /// Build a `Vec<json::Value>` that [`IFederationApi::request_raw`] expects when
 /// one argument are passed to the API call
 ///
 /// Notably the caling convention of fedimintd api is a bit weird ATM, so by
 /// using this function you'll make it easier to change it in the future.
-pub fn erased_single_param<Params>(param: &Params) -> Vec<JsonValue>
+pub fn erased_single_param<Params>(params: &Params) -> Vec<JsonValue>
```

Comment:
A little weird to call this `params` when the function title is `erased_single_param` :)

Reply:
Yeah I intend to do a follow-up refactor to clean this up, perhaps returning `ApiRequestErased`.